multiapps-controller-persistence/src/main/resources/org/cloudfoundry/multiapps/controller/persistence/db/changelog/db-changelog-I562548-1-persistence.xml

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.6.xsd">
+
+    <changeSet author="sap.com" id="add_columns_to_process_log">
+        <preConditions>
+            <tableExists tableName="process_log"/>
+        </preConditions>
+        <addColumn tableName="process_log">
+            <column name="id" type="VARCHAR(255)"/>
+        </addColumn>
+        <sql>
+            ALTER TABLE process_log ALTER COLUMN id DROP NOT NULL;
+        </sql>
+        <dropPrimaryKey tableName="process_log"/>
+        <addColumn tableName="process_log">
+            <column name="operation_log" type="VARCHAR"/>
+        </addColumn>
+        <sql>
+            ALTER TABLE process_log ALTER COLUMN file_id DROP NOT NULL;
+        </sql>
```

Comment:
Is this backwards compatible? We will drop the columns that might have logs, won't we lose the logs for the last 3 days?
What will happen if we need to revert? Did you test it?